docs/start/getting-started/fragments/flutter/generate-model.md

```diff
@@ -0,0 +1,27 @@
+With the basic setup complete, you can now model the data your application will work with. Amplify DataStore persists the modeled data on your local device and can even synchronize to a backend API without writing any additional code. These models are specified as [GraphQL](http://graphql.org/) schemas. You can [learn more](~/cli/graphql-transformer/overview.md)) about GraphQL schemas and data modeling. For now, let’s start by generating some initial data models.
+
+<amplify-callout>
+
+**Did you know?** You can get started with Amplify’s [admin UI](https://sandbox.amplifyapp.com/) without even creating an AWS Account.
+
+Try it out for yourself by following this tutorial's instructions for **Admin UI**.
+
+</amplify-callout>
+
+<amplify-block-switcher>
+
+<amplify-block name="Admin UI">
+
+<inline-fragment src="~/start/getting-started/fragments/flutter/blocks/generate-model-admin-ui.md"></inline-fragment>
+
+</amplify-block>
+
+<amplify-block name="CLI">
+
+<inline-fragment src="~/start/getting-started/fragments/flutter/blocks/generate-model-cli.md"></inline-fragment>
+
+</amplify-block>
+
+</amplify-block-switcher>
```

Comment:
I like this pattern for the Amplify documentation in general, but for the tutorial I believe we should be more opinionated and offer less options (ideally offer a single option). For the "Getting Started" of each category under the "Libraries" menu I think this pattern is the way to go, for the tutorial I'd say going with Admin UI only is perfectly reasonable.

pinging @renebrandel again, thoughts? I'm hesitant to make customers face a decision that they might not be able to this early in the process.

Reply:
I think it's a fair point to be opinionated, especially early on, and limit customer options to Admin UI. I was originally hesitant about making this change as it would be a large departure from our other platforms and so I opted for a more hybrid approach. But this, among other feedback, seems to indicate that maybe it's ok to just go ahead and offer what we feel is the recommended starting path. Our current block switching implementation also does not play as well with this pattern as it breaks the outline so that's another con for this hybrid approach.